browser/ui/views/toolbar/brave_toolbar_view.cc

```diff
@@ -122,6 +124,22 @@ void BraveToolbarView::OnLocationBarIsWideChanged() {
   SchedulePaint();
 }
 
+void BraveToolbarView::OnThemeChanged() {
+  ToolbarView::OnThemeChanged();
+
+  if (!brave_initialized_)
+    return;
+
+  if (display_mode_ == DisplayMode::NORMAL && bookmark_)
+    bookmark_->UpdateImage();
+}
+
+void BraveToolbarView::LoadImages() {
+  ToolbarView::LoadImages();
+  if (bookmark_)
```

Comment:
this doesn't seem right to me, what is the stack trace?

Reply:
```
[3784:38044:0703/102156.135:FATAL:flex_layout.cc(1006)] Check failed: it != child_params_.end().                                                                                              Backtrace:                                                                                                                                                                                            base::debug::CollectStackTrace [0x00007FFE79E625F0+48] (C:\bb3\brave-browser\src\base\debug\stack_trace_win.cc:284)                                                                           base::debug::StackTrace::StackTrace [0x00007FFE79E61760+80] (C:\bb3\brave-browser\src\base\debug\stack_trace.cc:206)                                                                          base::debug::StackTrace::StackTrace [0x00007FFE79E616E8+40] (C:\bb3\brave-browser\src\base\debug\stack_trace.cc:203)                                                                          logging::LogMessage::~LogMessage [0x00007FFE79EB0CAF+143] (C:\bb3\brave-browser\src\base\logging.cc:613)                                                                                      views::FlexLayout::GetFlexForView [0x00007FFE75E4F472+258] (C:\bb3\brave-browser\src\ui\views\layout\flex_layout.cc:1007)                                                                     views::internal::FlexLayoutInternal::CalculateNewLayout [0x00007FFE75E5244A+650] (C:\bb3\brave-browser\src\ui\views\layout\flex_layout.cc:754)                                                views::internal::FlexLayoutInternal::CalculateLayout [0x00007FFE75E50946+390] (C:\bb3\brave-browser\src\ui\views\layout\flex_layout.cc:425)                                                   views::FlexLayout::GetPreferredSize [0x00007FFE75E4FBD1+241] (C:\bb3\brave-browser\src\ui\views\layout\flex_layout.cc:1094)                                                                   views::FlexLayout::GetPreferredSize [0x00007FFE75E4FAC0+272] (C:\bb3\brave-browser\src\ui\views\layout\flex_layout.cc:1025)                                                                   views::View::CalculatePreferredSize [0x00007FFE75EA170F+111] (C:\bb3\brave-browser\src\ui\views\view.cc:1584)                                                                                 ToolbarView::CalculatePreferredSize [0x00007FFE6A0701AC+44] (C:\bb3\brave-browser\src\chrome\browser\ui\views\toolbar\toolbar_view.cc:558)                                                    views::View::GetPreferredSize [0x00007FFE75E97B43+131] (C:\bb3\brave-browser\src\ui\views\view.cc:388)                                                                                        ToolbarView::ChildPreferredSizeChanged [0x00007FFE6A070AC3+67] (C:\bb3\brave-browser\src\chrome\browser\ui\views\toolbar\toolbar_view.cc:645)                                                 views::View::PreferredSizeChanged [0x00007FFE75EA17B6+102] (C:\bb3\brave-browser\src\ui\views\view.cc:1595)                                                                                   views::LabelButton::PreferredSizeChanged [0x00007FFE75D50318+40] (C:\bb3\brave-browser\src\ui\views\controls\button\label_button.cc:447)                                                      views::LabelButton::ChildPreferredSizeChanged [0x00007FFE75D502E9+41] (C:\bb3\brave-browser\src\ui\views\controls\button\label_button.cc:442)                                                 views::View::PreferredSizeChanged [0x00007FFE75EA17B6+102] (C:\bb3\brave-browser\src\ui\views\view.cc:1595)                                                                                   views::ImageViewBase::PreferredSizeChanged [0x00007FFE75D76C9B+27] (C:\bb3\brave-browser\src\ui\views\controls\image_view_base.cc:146)                                                        views::ImageView::SetImage [0x00007FFE75D755FC+252] (C:\bb3\brave-browser\src\ui\views\controls\image_view.cc:43)                                                                             views::LabelButton::UpdateImage [0x00007FFE75D4DF9C+108] (C:\bb3\brave-browser\src\ui\views\controls\button\label_button.cc:356)                                                              views::LabelButton::SetImage [0x00007FFE75D4DF1C+76] (C:\bb3\brave-browser\src\ui\views\controls\button\label_button.cc:70)                                                                   BookmarkButton::SetToggled [0x00007FFE6824D07D+205] (C:\bb3\brave-browser\src\brave\browser\ui\views\toolbar\bookmark_button.cc:62)                                                           BookmarkButton::OnThemeChanged [0x00007FFE6824D17F+31] (C:\bb3\brave-browser\src\brave\browser\ui\views\toolbar\bookmark_button.cc:75)                                                        views::View::PropagateThemeChanged [0x00007FFE75E9D37F+271] (C:\bb3\brave-browser\src\ui\views\view.cc:2781)                                                                                  views::View::AddChildViewAtImpl [0x00007FFE75EA566C+1452] (C:\bb3\brave-browser\src\ui\views\view.cc:2169)                                                                                    views::View::AddChildViewAt [0x00007FFE682626AB+43] (C:\bb3\brave-browser\src\ui\views\view.h:429)                                                                            BraveToolbarView::Init [0x00007FFE6826234F+1023] (C:\bb3\brave-browser\src\brave\browser\ui\views\toolbar\brave_toolbar_view.cc:111)                                                          BrowserView::InitViews [0x00007FFE6A03BAD8+968] (C:\bb3\brave-browser\src\chrome\browser\ui\views\frame\browser_view.cc:2569)                                                                 BrowserView::ViewHierarchyChanged [0x00007FFE6A03B6FC+252] (C:\bb3\brave-browser\src\chrome\browser\ui\views\frame\browser_view.cc:2441)                                                      views::View::ViewHierarchyChangedImpl [0x00007FFE75EA5EBF+207] (C:\bb3\brave-browser\src\ui\views\view.cc:2315)                                                                               views::View::PropagateAddNotifications [0x00007FFE75EA60AC+268] (C:\bb3\brave-browser\src\ui\views\view.cc:2284)                                                                              views::View::AddChildViewAtImpl [0x00007FFE75EA5746+1670] (C:\bb3\brave-browser\src\ui\views\view.cc:2176)                                                                                    views::View::AddChildViewAt [0x00007FFE75EE231A+42] (C:\bb3\brave-browser\src\ui\views\view.h:429)                                                                         views::NonClientView::ViewHierarchyChanged [0x00007FFE75EE22B6+166] (C:\bb3\brave-browser\src\ui\views\window\non_client_view.cc:187)                                                         views::View::ViewHierarchyChangedImpl [0x00007FFE75EA5EBF+207] (C:\bb3\brave-browser\src\ui\views\view.cc:2315)                                                                               views::View::PropagateAddNotifications [0x00007FFE75EA60AC+268] (C:\bb3\brave-browser\src\ui\views\view.cc:2284)                                                                              views::View::AddChildViewAtImpl [0x00007FFE75EA5746+1670] (C:\bb3\brave-browser\src\ui\views\view.cc:2176)                                                                                    views::View::AddChildView [0x00007FFE75D2E149+57] (C:\bb3\brave-browser\src\ui\views\view.h:424)                                                                                 views::internal::RootView::SetContentsView [0x00007FFE75EC0711+337] (C:\bb3\brave-browser\src\ui\views\widget\root_view.cc:192)                                                               views::Widget::Init [0x00007FFE75EC6742+1506] (C:\bb3\brave-browser\src\ui\views\widget\widget.cc:355)                                                                                        BrowserFrame::InitBrowserFrame [0x00007FFE6B3B9F45+597] (C:\bb3\brave-browser\src\chrome\browser\ui\views\frame\browser_frame.cc:107)                                                         BrowserWindow::CreateBrowserWindow [0x00007FFE6B1DFDB2+594] (C:\bb3\brave-browser\src\chrome\browser\ui\views\frame\browser_window_factory.cc:27)                                             `anonymous namespace'::CreateBrowserWindow [0x00007FFE69F706C4+84] (C:\bb3\brave-browser\src\chrome\browser\ui\browser.cc:267)                                                                Browser::Browser [0x00007FFE69F6F817+3879] (C:\bb3\brave-browser\src\chrome\browser\ui\browser.cc:479)
...
```